.pre-commit-config.yaml

```diff
@@ -5,5 +5,5 @@ repos:
         name: pylint
         stages: [push]
         language: system
-        entry: pylint --disable=C0116 --ignore=venv
```

Comment:
@HelloFive Why were you using command? not the one from workflow?

Reply:
I didn't know there was example to follow